daos-cortx/docs/mpifileutils_setup.md

```diff
@@ -0,0 +1,430 @@
+        -- Set runtime path of "/root/mpifileutils_src/install/bin/dstripe" to "/root/mpifileutils_src/install/lib64:/usr/lib64/openmpi3/lib:/root/mpifileutils_src/install/lib:/root/daos_src/daos/install/lib64"
+        -- Installing: /root/mpifileutils_src/install/bin/dsync
+        -- Set runtime path of "/root/mpifileutils_src/install/bin/dsync" to "/root/mpifileutils_src/install/lib64:/usr/lib64/openmpi3/lib:/root/mpifileutils_src/install/lib:/root/daos_src/daos/install/lib64"
+        -- Installing: /root/mpifileutils_src/install/bin/dwalk
+        -- Set runtime path of "/root/mpifileutils_src/install/bin/dwalk" to "/root/mpifileutils_src/install/lib64:/usr/lib64/openmpi3/lib:/root/mpifileutils_src/install/lib:/root/daos_src/daos/install/lib64"
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dbcast.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dchmod.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dcmp.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dcp.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/ddup.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dbz2.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dfind.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dreln.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/drm.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dstripe.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dsync.1
+        -- Installing: /root/mpifileutils_src/install/share/man/man1/dwalk.1
+        [root@ssc-vm-2051 build]#
+
+
+# Object copy using mpifileutils
+
+* Create two empty POSIX containers using these steps[https://github.com/Seagate/cortx-experiments/blob/main/daos%2Bcortx/docs/setup_daos.md#start-agent]
+
+    - Pool creation & export
+    
+        Terminal log -
+            
+            [root@ssc-vm-2051 daos]# dmg -i pool create -s 1G
+            Creating DAOS pool with manual per-server storage allocation: 1.0 GB SCM, 0 B NVMe (100.00% ratio)
+            Pool created with 100.00% SCM/NVMe ratio
+            -----------------------------------------
+              UUID          : c7b0c9e2-028d-4dde-b016-a68743dba49a
+              Service Ranks : 0
+              Storage Ranks : 0
+              Total Size    : 1.0 GB
+              SCM           : 1.0 GB (1.0 GB / rank)
+              NVMe          : 0 B (0 B / rank)
+
+            [root@ssc-vm-2051 daos]#
+            [root@ssc-vm-2051 daos]#
+            [root@ssc-vm-2051 daos]# dmg -i system list-pools
+            Pool UUID                            Svc Replicas
+            ---------                            ------------
+            c7b0c9e2-028d-4dde-b016-a68743dba49a 0
+
+            [root@ssc-vm-2051 daos]# export pool1=c7b0c9e2-028d-4dde-b016-a68743dba49a
```

Comment:
use "source_pool_id" instead of pool1 and later "destination_pool_id" and similarly "source_container" and "destination_container" for better readability.

Reply:
done